src/Umbraco.Web.UI.Client/src/views/templates/edit.controller.js

```diff
@@ -559,11 +559,13 @@
                 }
             });
 
+            const currentTemplate = vm.templates.find(template => template.alias == vm.template.masterTemplateAlias);
```

Comment:
Maybe this could be moved down to line 582 so that it is kept together with the block where it is used?

Reply:
I have made a few adjustments :)